database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
Zijn migration bestanden nuttig, of kunnen die in de gitignore?

Reply:
Migrations willen we denk ik wel meenemen in de PRs. Dan kun je makkelijk iemand anders zijn local test data gebruiken.

Reply:
Migraties zijn cruciaal als je met meerdere mensen werkt met een database. Die zijn namelijk om database op te zetten en wijzingen te verwerken. Als er een wijzing moet worden gemaakt aan de database, dan doe je simpelweg een migratie aanmaken. Als andere projectleden dan aan het project werken hoeven zij alleen maar `php artisan migrate` uit te voeren. Hierdoor worden alleen migraties uitgevoerd die nieuw zijn. Daarom heeft iedere migratie ook een unieke bestandnaam, namelijk door de datum met tijd. Migraties kan je eenvoudig aanmaken door `php artisan make:migration`. 

Als je iemands testdata wilt gebruiken kun je het command `php artisan db:seed` gebruiken of als je voor de eerste keer migraties gaat uitvoeren ook `php artisan migrate:fresh --seed`.